### PR TITLE
Update va.gov build to latest version and new build method

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

See #1630 

* Update the vets website repo to the lastest version. This gets a reformated build script
* Update the `va:web:build` to use the `build-content` version of the build and pass in explicit API url.

---
This change is [Reviewable](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/1691)

### CODE REVIEW

**Comment by ElijahLynn** [comment]
Worth noting that @swirtSJW currently has a PR open to update the build that is failing consistently. #1639

File: composer.lock [hunk 1]
```diff
@@ -4116,9 +4116,8 @@
                     "2807629 - Allow menu items which link to unpublished nodes to be selected in the parent item selector": "https://www.drupal.org/files/issues/drupal-core-allow-menu-items-to-link-to-unpublished-2807629-7-8.3.x.patch",
                     "2893933 - claimItem in the database and memory queues does not use expire correctly": "https://www.drupal.org/files/issues/2020-01-07/2893933-34.patch",
                     "3057267 - User logout during maintenance mode": "https://www.drupal.org/files/issues/2019-09-11/3057267-maintenance-forced-logout-experimental.patch",
-                    "2775665 - MenuLinkContent updateLink function generates a PHP Warning for override-able keys that are not present in the loaded entity": "https://www.drupal.org/files/issues/updateLink-2775665-2.patch",
```
Comment (swirtSJW) [inline composer.lock:4119]: Was this removed intentionally?

**Comment by indytechcook** [comment]
Replacing with https://github.com/department-of-veterans-affairs/va.gov-cms/pull/1700

File: composer.lock (hunk 1 above)
Comment (ElijahLynn) [reply composer.lock:4119]: Yes, it actually made it in with the CEX merge but wasn't in composer.json, it was an in PR edit, and never actually was deployed. @oksana-c reviewed the patch more closely and was working on a better resolution to the issue.

**Pull request closed without merging**